python_training/config.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+OPERATIONAL_DB_FILE_NAME = 'midas.db'
+
+OPERATIONAL_DB = f'sqlite:///{OPERATIONAL_DB_FILE_NAME}'
+
+MEMBERS_INFO = [{'first_name': 'carmel', 'last_name': 'reubinoff', 'role': 'dev', 'location': 'Tozeret Haaretz'},
```

Comment:
Mock data should be saved in a dedicated file (`mock_data.py` or something similar).